core/themes/seven/css/style.css

```diff
@@ -755,16 +755,38 @@ a.button:active {
   text-decoration: none;
   color: #5a5a5a;
 }
-.node-form input#edit-submit,
-.node-form input#edit-submit-1 {
-  border: 1px solid #8eB7cd;
-  border-left-color: #8eB7cd;
-  border-right-color: #8eB7cd;
-  border-bottom-color: #7691a2;
-  background: url(../images/buttons.png) 0 -40px repeat-x;
-  color: #133B54;
+
+input.form-submit {
+  cursor: pointer;
+  padding: 10px 25px;
+  margin-bottom: 1em;
+  margin-right: 1em; /* LTR */
+  color: #444b53;
+  text-align: center;
+  text-transform: uppercase;
+  font-weight: normal;
+  font-size: 100%;
+  border-style: none;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
+  letter-spacing: .4px;
+  background-color: #e5e5e6;
+  transition: background-color 0.6s ease, color 0.6s ease;
+}
+input.form-submit:hover {
+  background-color: #CFDE56;
+  border-color: #CFDE56;
+}
+input.form-submit:active {
+  background-color: #666;
+  color: #fff;
+  text-shadow: #222 0 -1px 0;
 }
-input.form-submit:active,
+#edit-managed-file-upload {
+  margin: 0 5px 0 0;
+  padding: 9px;
```

Comment:
OK. Then would it be in order to make a change to the admin stylesheet for the file module where we currently have
``
.form-managed-file .form-file,
.form-managed-file .form-submit {
  margin: 0;
}
``